src/test/groovy/net/rubygrapefruit/platform/file/BasicFileEventFunctionsTest.groovy

```diff
@@ -514,34 +513,6 @@ class BasicFileEventFunctionsTest extends AbstractFileEventFunctionsTest {
         expectEvents change(CREATED, new File(reportedDir, fileInUppercaseDir.name))
     }
 
-    def "can handle exception in callback"() {
```

Comment:
In its current state, an interrupted exception can be thrown in the callback. We should test that this doesn't crash the JVM.

Reply:
That won't happen when we don't wait of queueing the events.